akka-actor/src/main/boilerplate/akka/japi/function/Functions.scala.template

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.function
+
+[3..22#/**
+ * A Function interface. Used to create 1-arg first-class-functions is Java.
+ * `Serializable` is needed to be able to grab line number for Java 8 lambdas.
+ */
+@SerialVersionUID(##1L)
+trait Function1[[#T1#], R] extends java.io.Serializable {
```

Comment:
right, now they look like:

    /**
     * A Function interface. Used to create 3-arg first-class-functions is Java.
     * `Serializable` is needed to be able to grab line number for Java 8 lambdas.
     */
    @SerialVersionUID(1L)
    trait Function3[-T1, -T2, -T3, +R] extends java.io.Serializable {
      @throws(classOf[Exception])
      def apply(arg1: T1, arg2: T2, arg3: T3): R
    }
    
    /**
     * A Function interface. Used to create 4-arg first-class-functions is Java.
     * `Serializable` is needed to be able to grab line number for Java 8 lambdas.
     */
    @SerialVersionUID(1L)
    trait Function4[-T1, -T2, -T3, -T4, +R] extends java.io.Serializable {
      @throws(classOf[Exception])
      def apply(arg1: T1, arg2: T2, arg3: T3, arg4: T4): R
    }

    /**
     * A Consumer interface. Used to create 2-arg consumers in Java.
     * A Procedure is like a Function, but it doesn't produce a return value.
     * `Serializable` is needed to be able to grab line number for Java 8 lambdas.
     */
    @SerialVersionUID(1L)
    trait Procedure2[-T1, -T2] extends java.io.Serializable {
      @throws(classOf[Exception])
      def apply(arg1: T1, arg2: T2): Unit
    }
    
    /**
     * A Consumer interface. Used to create 3-arg consumers in Java.
     * A Procedure is like a Function, but it doesn't produce a return value.
     * `Serializable` is needed to be able to grab line number for Java 8 lambdas.
     */
    @SerialVersionUID(1L)
    trait Procedure3[-T1, -T2, -T3] extends java.io.Serializable {
      @throws(classOf[Exception])
      def apply(arg1: T1, arg2: T2, arg3: T3): Unit
    }